OTAnalytics/plugin_video_processing/video_reader.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+import numpy as np
+from moviepy.video.io.VideoFileClip import VideoFileClip
+
+from OTAnalytics.application.datastore import VideoReader
+
+
+class FrameDoesNotExistError(Exception):
```

Comment:
Should we extract it to another module? If so which layer should it belong to?

Reply:
I would place it here for the moment. It might move into the application code (`datastore.py`).